Example texturefiltering crashing [bug]

### What happend?

When trying to run example texturefiltering, it crashes with:
```
$ go run texturefiltering.go
2022/04/06 22:50:04 giu: .NewTextureFromRgba: cannot load texture befor (*MasterWindow).Run call!
panic: giu: .NewTextureFromRgba: cannot load texture befor (*MasterWindow).Run call!

goroutine 1 [running, locked to thread]:
log.Panicf({0x7ff65bb8e440?, 0x33?}, {0xc000115e90?, 0x0?, 0x0?})
        C:/Bin/Go/src/log/log.go:392 +0x67
github.com/AllenDang/giu.fatal({0x0?, 0x7ff65b7c2f00?}, {0x7ff65bb907ba, 0x12}, {0x7ff65bb9fe4d?, 0xc0000e4cb0?}, {0x0?, 0xc0000c6060?, 0xc0000b0200?})
        C:/Workspace/go/github/giu/Utils.go:208 +0x1a5
github.com/AllenDang/giu.Assert(...)
        C:/Workspace/go/github/giu/Utils.go:199
github.com/AllenDang/giu.NewTextureFromRgba({0x7ff65bbdc3f0?, 0xc0000aa100?}, 0x320?)
        C:/Workspace/go/github/giu/Texture.go:38 +0x65
main.main()
        C:/Workspace/go/github/giu/examples/texturefiltering/texturefiltering.go:88 +0x85
exit status 2
```

### Code example

code is in example texturefiltering

### To Reproduce

1- go run texturefiltering.go

### Version

master

### OS

windows 11 64 bits

--- a/Texture.go
+++ b/Texture.go
@@ -35,7 +35,6 @@ func EnqueueNewTextureFromRgba(rgba image.Image, loadCb func(t *Texture)) {
 
 // NewTextureFromRgba creates a new texture from image.Image and, when it is done, calls loadCallback(loadedTexture).
 func NewTextureFromRgba(rgba image.Image, loadCallback func(*Texture)) {
-	Assert(Context.isRunning, "", "NewTextureFromRgba", "cannot load texture befor (*MasterWindow).Run call!")
 	loadTexture(rgba, loadCallback)
 }
 

